packages/plugin-js/package.json

```diff
@@ -34,7 +34,7 @@
     "test:ts": "tsc --noEmit"
   },
   "peerDependencies": {
-    "@terrazzo/cli": "^0.2.0"
+    "@terrazzo/cli": "^0.3.0"
```

Comment:
Doesn’t need a release technically because this plugin doesn’t consume any of this code. Just for the next breaking release